hack/bin/oauth_test.sh

```diff
@@ -64,13 +64,17 @@ AUTH_CODE=$(
     awk -F ': ' '/^Location/ {print $2}' | awk -F'[=&]' '{print $2}'
 )
 
-ACCESS_TOKEN=$(
-  curl -s -X POST localhost:8082/oauth/token \
+ACCESS_TOKEN_RESPONSE=$(
+  curl -s -X POST localhost:8080/oauth/token \
     -H "Content-Type: application/x-www-form-urlencoded" \
-    -d 'code='"$AUTH_CODE"'&client_id='"$CLIENT_ID"'&grant_type=authorization_code&redirect_uri=https://example.com&client_secret='"$CLIENT_SECRET" |
-    jq -r '.accessToken'
+    -d 'code='"$AUTH_CODE"'&client_id='"$CLIENT_ID"'&grant_type=authorization_code&redirect_uri=https://example.com&client_secret='"$CLIENT_SECRET"
 )
 
+echo "$ACCESS_TOKEN_RESPONSE" | jq
```

Comment:
```suggestion
echo "$ACCESS_TOKEN_RESPONSE" | jq >&2
```

ya, i'm nit-picking again :)